parser/ParseStatement.hpp

```diff
@@ -562,6 +566,43 @@ class ParseStatementInsert : public ParseStatement {
     return relation_name_.get();
   }
 
+ private:
+  std::unique_ptr<ParseString> relation_name_;
```

Comment:
Why adding a `private` member in the base class? It is not visible to its derived classes.